### ISSUE TITLE
commentedOutCode: don't warn on "Output:" comments

### ISSUE THREAD

**Issue by quasilyte** [go-critic/go-critic#924]
```go
	// Output:
	// x = 1
```
This style of comments is used in example tests.

**Comment by quasilyte** [comment]
Looks like it's not an issue anymore. We have "Output:" special handling implemented.
